apps/aechannel/src/aesc_codec.hrl

```diff
@@ -18,27 +19,32 @@
 -define(WDRAW_LOCKED , withdraw_locked).
 -define(WDRAW_ERR    , withdraw_error).
 -define(ERROR        , error).
+-define(LEAVE        , leave).
+-define(LEAVE_ACK    , leave_ack).
 -define(SHUTDOWN     , shutdown).
 -define(SHUTDOWN_ACK , shutdown_ack).
 -define(INBAND_MSG   , inband_msg).
 
 -define(ID_CH_OPEN      , 1).
 -define(ID_CH_ACCEPT    , 2).
 -define(ID_CH_REESTABL  , 3).
--define(ID_FND_CREATED  , 4).
--define(ID_FND_SIGNED   , 5).
--define(ID_FND_LOCKED   , 6).
--define(ID_UPDATE       , 7).
--define(ID_UPDATE_ACK   , 8).
--define(ID_UPDATE_ERR   , 9).
--define(ID_DEP_CREATED  , 10).
--define(ID_DEP_SIGNED   , 11).
--define(ID_DEP_LOCKED   , 12).
--define(ID_DEP_ERR      , 13).
--define(ID_WDRAW_CREATED, 14).
--define(ID_WDRAW_SIGNED , 15).
--define(ID_WDRAW_LOCKED , 16).
--define(ID_WDRAW_ERR    , 17).
+-define(ID_CH_REEST_ACK , 4).
+-define(ID_FND_CREATED  , 5).
+-define(ID_FND_SIGNED   , 6).
+-define(ID_FND_LOCKED   , 7).
+-define(ID_UPDATE       , 8).
+-define(ID_UPDATE_ACK   , 9).
+-define(ID_UPDATE_ERR   , 10).
+-define(ID_DEP_CREATED  , 11).
+-define(ID_DEP_SIGNED   , 12).
+-define(ID_DEP_LOCKED   , 13).
+-define(ID_DEP_ERR      , 14).
+-define(ID_WDRAW_CREATED, 15).
+-define(ID_WDRAW_SIGNED , 16).
+-define(ID_WDRAW_LOCKED , 17).
+-define(ID_WDRAW_ERR    , 18).
+-define(ID_LEAVE        , 94).
```

Comment:
Does this require spec at https://github.com/aeternity/protocol/blame/52de7f9a1e86428a9d6b729d72764f088372c440/channels/OFF-CHAIN.md ?